controllers/phish.go

```diff
@@ -112,6 +112,7 @@ func (ps *PhishingServer) registerRoutes() {
 	router.PathPrefix("/static/").Handler(http.StripPrefix("/static/", fileServer))
 	router.HandleFunc("/track", ps.TrackHandler)
 	router.HandleFunc("/robots.txt", ps.RobotsHandler)
+	router.HandleFunc("/arbevent", ps.ArbitraryEventHandler)
```

Comment:
Nit: Let's rethink this path. Something a bit cleaner. I'd be fine with `/event`, `/custom_event`, etc.

I know that there's a risk for `/event` to be used in the phishing URL, so that might be something to consider. Though, I don't think it's a _huge_ risk.

Reply:
Changed to "/event"